target/linux/ath79/image/generic.mk

```diff
@@ -308,6 +308,17 @@ define Device/comfast_cf-wr650ac-v2
 endef
 TARGET_DEVICES += comfast_cf-wr650ac-v2
 
+define Device/compex_wpj531-16m
+  ATH_SOC := qca9531
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
done ;-)